src/Mimi.jl

```diff
@@ -14,50 +14,32 @@ using StringBuilders
 
 export
     @defcomp,
-    @modelegate,
     MarginalModel,
     Model,
-    addcomponent,
-    add_connector_comps,
-    add_dimension!,
-    addparameter,
-    compdef,
-    compdefs,
-    compinstance,
-    compkeys,
+    addcomponent,  # push!, add_component!, add_component, addcomp!, add!; could also have multiple functions like insert!, push! etc. instead of before after keywords
```

Comment:
suggestion:  `add_comp`
reason:  matches `_` convention; matches using `comp, var, and param`; `add!` seems perhaps too simplified for now since we generally do specific comp, var, or param

Reply:
I agree.